### PR TITLE
docker:go: add option to skip runtime image.

### PR DESCRIPTION
**Pull request by raulk**
For builds that do more than just compiling go code (e.g. generating things, linking to dynamic libs, etc.), we might want to just use the build container as our runtime container. This is suboptimal, but it's a good way to bootstrap a test plan for a complex target, like Lotus/Filecoin.

### CODE REVIEW

File: pkg/build/docker_go.go [hunk 1]
```diff
@@ -424,4 +431,6 @@ COPY --from=builder /plan/testplan /
 
 EXPOSE 6060
 ENTRYPOINT [ "/testplan"]
+
+{{ end }}
```
Comment (nonsense) [inline pkg/build/docker_go.go:435]: I think this should be above `EXPOSE 6060`.

Comment (nonsense) [reply pkg/build/docker_go.go:435]: You also need:
```
{{ if .SkipRuntimeImage }}
RUN cp /testground_dep_list /
RUN /plan/testplan /
{{ end }}
```

Comment (raulk) [reply pkg/build/docker_go.go:435]: Oh yeah, absolutely. And we need to put the test plan binary at the root.

Comment (raulk) [reply pkg/build/docker_go.go]: Achieved the symmetry but differently: 

[https://github.com/testground/testground/pull/1078/commits/548da321a79cc807d1689fc69f1beec079873371 ](https://github.com/testground/testground/pull/1078/commits/548da321a79cc807d1689fc69f1beec079873371)

**Pull request merged**